### PR TITLE
Add some debugging.

### PR DESCRIPTION
**Pull request by ashleypittman**

### CODE REVIEW

File: check_python.sh [hunk 1]
```diff
@@ -52,14 +52,21 @@ if [ -n "${project_check_module}" ]; then
   cm_pylint_out="${PWD}/pylint.log"
   rm -f "${cm_pylint_out}"
   # Must suppress issues being written to stdout.
-    "${project_check_module}" > check_module.out
-    if [ -s "${cm_pylint_out}" ]; then
+  "${project_check_module}" > check_module.out
+  rc=$?
```
Comment (brianjmurrell) [inline check_python.sh]: I prefer `${PIPESTATUS[0]}` as it's future-proof to somebody adding `| <some_filter>` to the command without noticing they are altering the checking of the status on `${project_check_module}`.
```suggestion
  rc=${PIPESTATUS[0]}
```

File: check_python.sh [hunk 2]
```diff
@@ -52,14 +52,21 @@ if [ -n "${project_check_module}" ]; then
...
       grep -E ".+:[[:digit:]]+:.+:.+" "${cm_pylint_out}"
       popd || exit 1
       exit 1
-    else
+  else
       rm -f "${cm_pylint_out}"
-    fi
+  fi
+  if [ $rc != 0 ]; then
+      echo "Critical error running ${cm_pylint_out}"
```
Comment (brianjmurrell) [inline check_python.sh]: _`running  ${cm_pylint_out}`_?  ${cm_pylint_out} looks like an output file.

File: check_python.sh [hunk 3]
```diff
@@ -65,7 +65,7 @@ if [ -n "${project_check_module}" ]; then
       echo "Critical error running ${project_check_module}"
       echo command output was
       cat check_module.out
-      exit ${rc}
+      exit 1
```
Comment (brianjmurrell) [inline check_python.sh]: The lint fix was just to quote `${rc}` wasn't it?

Comment (ashleypittman) [reply check_python.sh]: The lint fix was to quote $rc, I thought adding the {} would fix it but obviously not.  We're not actually trying to convey any information thought the code, just a pass/fail so this is just as reliable.

File: github_checkpatch.py [hunk 4]
```diff
@@ -330,6 +331,7 @@ class Reviewer(object):
     """
     def __init__(self):
         self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.DEBUG)
         self.project, self.repo = os.environ['GIT_URL'].split('/')[-2:]
```
Comment (ashleypittman) [inline github_checkpatch.py:335]:
```suggestion
    def __init__(self):
        self.logger = logging.getLogger(__name__)
        self.logger.setLevel(logging.DEBUG)

        self.project, self.repo = os.environ['GIT_URL'].split('/')[-2:]
```

**Pull request closed without merging**